ansible/roles/ssc/templates/start.sh

```diff
@@ -12,6 +18,57 @@ for VAR in $REQUIRED_VARS; do
   fi
 done
 
+# Check if we should start from state sync
+ShouldStartFromStateSync() {
+  if [ -z "$SYNC_RPC" ]; then
```

Comment:
Shall we also pass STATE_SYNC_ENABLED? So, we have a kill switch. Probably cleaner.